libs/ui-components/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export * from './types/types'
 export * from './button'
 export * from './color-bar'
-export * from './countryflag'
+// export * from './countryflag'
```

Comment:
you found issues with this library, didn't you? Something I need to review why

Reply:
Yes, it was causing errors when building / starting the next app. We'll have to review it